src/NWCClient.ts

```diff
@@ -687,14 +715,14 @@ export class NWCClient {
     let subscribed = true;
     let endPromise: (() => void) | undefined;
     let onRelayDisconnect: (() => void) | undefined;
-    let sub: Sub<23196> | undefined;
+    let sub: Sub<number> | undefined;
     (async () => {
       while (subscribed) {
         try {
           await this._checkConnected();
           sub = this.relay.sub([
             {
-              kinds: [23196],
+              kinds: [...(this.supportedVersion ? [23196] : [23197])],
```

Comment:
This is wrong, if supported version exists and is "0.0", it should be 23196, not 23197